### PR TITLE
Team and headless doc updates

### PR DESCRIPTION
**Pull request by cb1kenobi**

### CODE REVIEW

File: content/en/docs/authentication/_index.md [hunk 1]
```diff
@@ -128,7 +138,11 @@ axway auth logout [options] [<accounts...>]
 
 ### switch
 
-Once authenticated into at least one account, you can set the default account and organization to use for `axway` commands.
+Once authenticated, you can set the default account, organization, and current team to use for `axway` commands.
+
+Only platform accounts have an organizations. If the selected account is a service account, then the organization selection is skipped.
```
Comment (sallylanda6) [inline content/en/docs/authentication/_index.md:143]: Only platform accounts have an organization (fix typo)?

**Review by sallylanda6 — CHANGES_REQUESTED** [review]
One small correction is needed. Otherwise, it looks great!

**Review by sallylanda6 — APPROVED** [review]
Looks good! I'll get the docs published.

**Comment by cb1kenobi** [comment]
Thank you @sallylanda6!

**Pull request merged**